src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
don't reset `margin` for all elements(`*`)

Reply:
just set `margin: 0` for exact elements you use

Reply:
checklist
<img width="884" alt="image" src="https://user-images.githubusercontent.com/102411130/218027937-266814b2-ca1e-4643-a044-c05028ab49d3.png">